File: MODUL-5/python/PRAK502-2310817210007-RAYMONDHARIYONO.py
import math

def mutlak(angka):
    return -angka if angka < 0 else angka

def hitung(nilai1, nilai2):
    return mutlak(nilai1 - nilai2)

a, c, b, d = map(int, input().split())


Hasil = hitung(a, b) + hitung(c, d)
print(mutlak(Hasil))

File: MODUL-5/c/PRAK502-2310817210007-RAYMONDHARIYONO.c
#include <stdio.h>
#include <math.h>

int mutlak(int angka){
    return (angka < 0) ? -angka: angka;
}

int hitung(int nilai1, int nilai2){
    return mutlak(nilai1 - nilai2);
}

int main()
{
 int a,b,c,d,Hasil;

 scanf("%d%d%d%d",&a,&c,&b,&d);

 Hasil = hitung(a,b) + hitung(c,d);
 printf("%d",mutlak(Hasil));
 return 0;
}